ApiHelper.php

```diff
@@ -173,57 +200,13 @@ public function getMonthlyPageviews( $pages, $start, $end ) {
 				}
 			}
 			unset( $responses );
-			gc_collect_cycles();
```

Comment:
Is this no longer useful? I think we found a need to do this when we were writing the bot code in the early stages but I forget exactly why.

Reply:
I think we did it as an experiment, that didn't work. From my understanding PHP is that smart these days, you shouldn't need to do manual garbage collection. We aren't processing WikiProject Biography anymore which was the one that was failing.